chart/templates/deployment.yaml

```diff
@@ -194,9 +194,14 @@ spec:
       serviceAccountName: {{ include "vso.chart.fullname" . }}-controller-manager
       securityContext:
         {{- toYaml .Values.controller.podSecurityContext | nindent 8 }}
+      {{- with .Values.controller.imagePullSecrets }}
+      imagePullSecrets:
+      {{- toYaml . | nindent 8 }}
+      {{- end }}
```

Comment:
The image pull secrets are already set on the deployment's service account (line 14), so I don't think this is necessary.

Reply:
Reverted in c1dfa0095f554f9f65e94605cbcbf0e2ea0b71a3